frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -686,8 +697,10 @@
                 for (var j=0; j<vm.phases.results.length; j++){
                     var offset = new Date(vm.phases.results[j].start_date).getTimezoneOffset();
                     vm.phases.results[j].start_zone = moment.tz.zone(timezone).abbr(offset);
```

Comment:
Actually, you were right, let's just use one variable for timezone and gmt zone. No need to have separate start and end zone. I assume `moment.tz.guess` will return same thing at both places.